mbed_app.json

```diff
@@ -1,5 +1,9 @@
 {
     "config": {
+	"wifi-shield": {
+            "help": "Options are WIFI_ESP8266, WIFI_IDW01M1",
+            "value": "WIFI_IDW01M1"
+	},
         "wifi-ssid": {
```

Comment:
@betzw Can we add an entry in the 'target_overrides' section in this file to get the NUCLEO_F401RE working out of the box with D8, D2?

Reply:
Done.